satpy/etc/composites/ami.yaml

```
@@ -137,11 +137,11 @@ composites:
     compositor: !!python/name:satpy.composites.SelfSharpenedRGB
     prerequisites:
       - name: NR016
-        modifiers: [sunz_corrected] #, rayleigh_corrected]
+        modifiers: [sunz_corrected]
       - name: VI008
-        modifiers: [sunz_corrected] #, rayleigh_corrected]
+        modifiers: [sunz_corrected]
       - name: VI006
-        modifiers: [sunz_corrected] #, rayleigh_corrected]
+        modifiers: [sunz_corrected, rayleigh_corrected]
```

```suggestion
        modifiers: [sunz_corrected]
```
I think it is best not to include the correction at this point. But to be considered.